portfolio/src/main/java/com/google/sps/servlets/CommentsServlet.java

```diff
@@ -50,8 +52,10 @@
     private static final String MAX_COMMENTS_PARAMETER = "maxComments";
     private static final String EMAIL_PARAMETER = "email";
     private static final String ID_PARAMETER = "id";
+    private static final String NICKNAME_PARAMETER = "nickname";
     private static String maxComments;
     private static final UserService userService = UserServiceFactory.getUserService();
+    private static final User user = userService.getCurrentUser();
```

Comment:
put up with the other `private static final` variables. Also, these should be named `USER_SERVICE` and `USER`